### PR TITLE
[tests-only][full-ci] refactoring user remove from group test code 

### PR DESCRIPTION
**Pull request by S-Panta**
## Description
This PR refactors the user removal to group test code from ocs api to graphapi

## How Has This Been Tested?

- test environment:
- locally
- ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 1]
```diff
@@ -360,14 +360,16 @@ public function adminDeletesUserUsingTheGraphApi(string $user, ?string $byUser =
...
+	 * @param string $group
+	 * @param string $user
 	 * @param string|null $byUser
 	 *
 	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public function removeUserFromGroup(string $groupId, string $userId, ?string $byUser = null): ResponseInterface {
+	public function removeUserFromGroup(string $group, string $user, ?string $byUser = null): ResponseInterface {
+		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id") ?: WebDavHelper::generateUUIDv4();
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/bootstrap/GraphContext.php:371]:
```suggestion
		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id") ?? WebDavHelper::generateUUIDv4();
```

Comment (S-Panta) [reply tests/acceptance/features/bootstrap/GraphContext.php:371]: `$this->featureContext->getAttributeOfCreatedGroup($group, "id")` generate false when group is given as `empty string`  so, ?: is used

Comment (saw-jan) [reply tests/acceptance/features/bootstrap/GraphContext.php:371]: `?:` is the short form of ternary operator in php

**Review by saw-jan — APPROVED** [review]
Looks good. please, try reva bump

**Review by saw-jan — CHANGES_REQUESTED** [review]
Need to add GraphContext to one of the suites.
https://drone.owncloud.com/cs3org/reva/3499/12/6

**Review by saw-jan — APPROVED** [review]
I think we can merge it now.

**Pull request merged**